docs/blog/2019-09-11-gazette-september/index.mdx

```diff
@@ -0,0 +1,119 @@
+The Gatsby team has had a busy month. We've made some exciting improvements.
+
+### Accessibility
+
+[Marcy Sutton](https://twitter.com/marcysutton) from Gatsby's Learning team has been doing some incredible work in making Gatsby more accessible by default.
+
+A big part of making the web more accessible is teaching us how to write more accessible apps! Back in June, Marcy conducted some user testing sessions for accessibility research. She wrote about it in her post, [What we learned from user testing of accessible client-side routing techniques with Fable Tech Labs](https://www.gatsbyjs.org/blog/2019-07-11-user-testing-accessible-client-routing/) which I consider canonical and _required_ reading.
+
+Marcy also presented a full-length workshop on [Accessibility in JavaScript Applications for Frontend Masters](https://frontendmasters.com/workshops/javascript-accessibility/), including some exclusive tidbits about building accessible Gatsby sites. In case you haven't read her post or seen her workshop already, please _do_ to learn how to make your web apps more accessible.
+
+### GraphQL
+
+We've made a bunch of improvements to Gatsby's GraphQL layer recently. Here is a quick summary:
+
+- We now support abstract data interfaces. This means that we can now build Gatsby sites (or themes) that support the same query for multiple data sources. [Mikhail](https://twitter.com/freiksenet) from the core team talks more about this on [Learn With Jason](https://www.youtube.com/watch?v=ALspNtrOqDk)
+- Materialisation
+- [Stefan](https://github.com/stefanprobst) added the ability to snapshot your Gatsby schema in [gatsby-plugin-schema-snapshot](https://www.gatsbyjs.org/packages/gatsby-plugin-schema-snapshot/). This enables you lock-down a project’s GraphQL schema.
+- GraphiQL Explorer is now available in WPGraphQL as well
+
+### gatsby-plugin-offline v3
+
+[David](https://twitter.com/davidbailey00) (who recently joined the Gatsby core team) has been working on [gatsby-plugin-offline](https://www.gatsbyjs.org/packages/gatsby-plugin-offline/) for a while and recently released v3 of the plugin.
+
+v3 includes an updated Workbox, the ability to append custom scripts to the generated service worker and many more smaller improvements.
+
+### Optimized JavaScript bundle configuration
+
+[Ward](https://twitter.com/wardpeet) from the core team recently made some improvements to Gatsby's webpack config in [#17093](https://github.com/gatsbyjs/gatsby/pull/17093). This introduces a better splitChunks config and results in a 400 KB reduction in initial bundle size and a 400 ms faster [TTI](https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive) for [gatsbyjs.org](https://gatsbyjs.org).
+
+Huge shoutout to the Next.js team since we picked this up from them!
+
+### Renovate
+
+We recently enabled automatic dependency updates using [Renovate](https://renovatebot.com/). As of today, dependencies in all official Gatsby packages are completely up to date.
```

Comment:
Including major versions?

Reply:
Good point. Minor and patch only. Will mention that in parentheses.